src/generate/array_proxy.rs

```diff
@@ -0,0 +1,46 @@
+
```

Comment:
Add
```
#![allow(unused)]
```

Reply:
Isn't `allow(unused)` redundant on `pub` definitions? E.g., nothing in `generic.rs` currently has `allow(unused)`?
Or am I missing something?